### PR TITLE
Upgrade to 1.5.2

### PR DESCRIPTION
**Pull request by tituspijean**
- Install and update zip have been updated to version 1.5.2
- After upgrade `bin/gpm update -y` command is started to update all the plugins

### CODE REVIEW

File: scripts/upgrade [hunk 1]
```diff
@@ -103,6 +103,16 @@ ynh_system_user_create "$app"
...
 
+#=================================================
+# SPECIFIC UPGRADE
+#=================================================
+# UPDATE PLUGINS
+#=================================================
+
+pushd "$final_path"
+	exec_as "$app" "bin/gpm update -y"
+popd
```
Comment (Jibec) [inline scripts/upgrade]: I'm not found of this, pushd/popd, it works really fine, but I find it difficult to use for newcomers. Using a sub-shell and having a `cd` gives the same result and is easier to read (for newcomers).

**Comment by Jibec** [comment]
so, the gpm update breaks the upgrade, so we should not use it :(
in addition, master and upgrade does not install anymore on YunoHost 3.2.2 (only Tried my VM)... no idea why:

> Whoops \ Exception \ ErrorException (E_WARNING)
> session_write_close(): open(/var/lib/php/sessions/sess_apeem5vufm5jgr79eii5pok060, O_RDWR) failed: Permission denied (13)

I assume one of these commits broke something: https://github.com/YunoHost/yunohost/commits/stretch-stable/data/helpers.d/backend

File: scripts/upgrade (hunk 1 above)
Comment (tituspijean) [reply scripts/upgrade]: No problem :)

```sh
exec_as "$app" "cd "$final_path" && bin/gpm update -y"
```
This should be the alternative.

**Comment by tituspijean** [comment]
I think these errors happen because we try to perform the update before running `chown` again. Let's try to swap these commands places.

**Comment by Jibec** [comment]
The gpm command still promps for an input from the user, resulting an infinite waiting in the admin interface. While I understand the aim, I think it's better to revert this.

**Comment by tituspijean** [comment]
Damn that is weird. The `-y` flag is designed to avoid any prompt, and I got none during my tests.

**Comment by Jibec** [comment]
it's because now we have version "1.5.3" available, so grav find himself smart to warn us ;)

**Comment by tituspijean** [comment]
OK, I yield. We cannot win against commands that do more than what they are documented to. :)
I propose to directly upgrade to the newest version before merging, no need to stay to much outdated.

**Comment by Jibec** [comment]
oops, anyway, the next PR will be faster ;)

**Pull request merged**